rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -111,17 +111,17 @@ components:
             $ref: '#/components/schemas/ErrorMessage'
 
   parameters:
-    Keyword:
+    QueryString:
       name: q
       in: query
       schema:
         type: string
         minLength: 1
         maxLength: 100
         pattern: .*\S.*
-        example: "house"
+        example: "contents LIKE 'metadata contents'"
```

Comment:
✏️ Change this example to `title` instead of `contents` because we're pretty sure we'll support title at the end of the day, not contents